src/frontend/src/lib/types/onramper.ts

```diff
@@ -0,0 +1,3 @@
+// The list of networks that are supported by OnRamper can be found here:
+// https://docs.onramper.com/docs/network-support
+export type OnRamperNetworkId = string;
```

Comment:
Can we list only the subset of those we are using instead of `string`?

Reply:
Better! Changed